action/template.go

```diff
@@ -28,8 +28,12 @@ func GenerateTemplate(out io.Writer, in io.Reader, vals interface{}) {
 }
 
 // Template renders a template to an output file.
-func Template(out, in, data string) {
+func Template(out, in, data string, force bool) {
 	var dest io.Writer
+	_, err := os.Stat(out)
+	if !force && err == nil {
+		log.Die("re-run the command with -f flag")
```

Comment:
Why? We should probably explain more clearly to the user why they need to run with the --force flag.